client/gefyra/cluster/resources.py

```diff
@@ -159,6 +160,28 @@ def create_operator_deployment(
     return deployment
 
 
+def _check_pod_for_command(pod: V1Pod, index=0):
+    containers: list[V1Container] = pod.spec.containers
+    if not len(containers):
+        raise RuntimeError("No container available in pod {pod.metadata.name}.")
+
+    if containers[index].command:
```

Comment:
`command` is not always a problem. https://gefyra.dev/details/what-is-gefyra/#gotchas--current-limitations says
"other than common shells". `sh`, `bash` would be ok. maybe `ash` or `zsh` (to be tested with `docker run -it --entrypoint <shell> gefyra-carrier "echo blubb"` (or similar, not tested tho).

Reply:
I can confirm working: `sh`, `bash`, `zsh`, `ash`.
`docker run --rm --entrypoint bash quay.io/gefyra/carrier echo "blubb"` (still starts nginx)

Reply:
Added check for given commands :)